client/storage-monitor/src/lib.rs

```diff
@@ -45,19 +45,19 @@ pub struct StorageMonitorParams {
 	#[arg(long = "db-storage-threshold", value_name = "MB", default_value_t = 1000)]
 	pub threshold: u64,
 
-	/// How often available space is polled.
+	/// How often available space is polled in seconds.
 	#[arg(long = "db-storage-polling-period", value_name = "SECONDS", default_value_t = 5, value_parser = clap::value_parser!(u32).range(1..))]
```

Comment:
I think the default `5` is not good. `6` or `12` is better idea.

Reply:
Why 6 instead of 5?

Reply:
Personally, I think storage consumption keeps a rhythm with block generation time, so 6 or 12 is better, after all, there is currently no substrate chain with 5s block generation